src/scripts/towns/PokemonContest.ts

```diff
@@ -7,7 +7,7 @@ class PokemonContest implements Feature {
 
     public prizes = [
         new PokemonContestPrizes('10 Rare Candy',
-            'Get 10 Rare Candy by catching a Machop! (real requirement will be added later)',
+            'Get 10 Rare Candy by catching a entering a contest! (real requirement will be added later)',
```

Comment:
this still says "catching a"

Reply:
changed this meaningless text, real requirement will still be added later

Reply:
I just wanted to point it out because it's now gramatically incorrect compared to before, better safe than sorry (although I'm sure someone else would've casugt it before official release, sorry)